package-lock.json

```diff
@@ -6,7 +6,7 @@
 	"packages": {
```

Comment:
I don't see any reason why the lock file needs to be altered?

Reply:
Well, that automatically happens if the versions in the `package.json` are not fixed (everything is using `^` = "Compatible with version" according to [semver] - see [NPM docs](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#dependencies)) and new versions matching that range are out. This means it simply pulled.

**Example**: TypeScript was updated from 5.1.2 to 5.2.2 because either one of `microbundle` or `eslint-config-developit` is not using fixed versions for dependencies and newer versions are out between the last `npm install` to today. Same rules apply to this project. 

If you don't want this to change I'd need to lock the versions of all deps 😃 Just let me know if I should do that.

Reply:
Yes, I'm aware of how a `package-lock.json` works. The point is that unless there's a reason for the change, such as needing a newer version of a dependency, it generally should not be touched. This applies for all files, in that keeping the diff and blame to a minimum helps keep thing sane over the years. That you used `npm install` over `npm ci` is not reason enough to change the lock file, IMO.

You don't need to lock the deps, just don't add it to the commit or use `npm ci`.

Reply:
Ahhh I completely misunderstood your input, sorry my bad!

Reply:
Updated microbundle to v0.15.1